web/package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "precommit": "lint-staged",
     "lint": "eslint src",
-    "start": "REACT_APP_BACKEND_SERVICE=http://192.168.19.96:30020/api/v1 node scripts/start.js",
+    "start": "REACT_APP_API_BASE_URL=/api/v1 node scripts/start.js",
```

Comment:
How about change to: `API_BASE_URL` ?

Reply:
the `REACT_APP_` prefix is appointed by https://github.com/caicloud/cyclone/blob/develop/web/config/env.js#L58